### PR TITLE
added copyRelease [WIP]

### PR DESCRIPTION
**Pull request by fceller**

### CODE REVIEW

**Comment by Luschgy** [comment]
Is this function only for .deb & .rpm packages ?

File: helper.linux.fish [hunk 1]
```diff
@@ -737,5 +745,64 @@ function makeSnippets
   and buildTarGzSnippet
 end
 
+function copyReleaseEdition
+  set -l dest $argv[1]
+  set -l repo $argv[2]
+  set -l edition $argv[3]
+  set -l l $argv[4]
+
+  # debian
```
Comment (dothebart) [inline helper.linux.fish]: shouldn't there be a central place where we calculate the filenames?

Comment (fceller) [reply helper.linux.fish]: Good point, will add

**Comment by fceller** [comment]
@Luschgy it should later also apply to tar.gz & Mac & Windows. Still need to find out how to do it. Maybe we can have a look tomorrow together.

**Pull request closed without merging**